src/pages/components/pane.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Pane
+description: >
+  Panes give users a quick way to resize layouts vertically or horizontally by
+  interacting with the splitter. They can also be used to create expandable or
+  collapsible layouts.
+package: '@zendeskgarden/react-grid'
+components:
+  - grid/src/elements/pane/PaneProvider.tsx
+  - grid/src/elements/pane/Pane.tsx
+subcomponents:
+  - grid/src/elements/pane/components/Content.tsx
+  - grid/src/elements/pane/components/Splitter.tsx
+  - grid/src/elements/pane/components/SplitterButton.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To make a structured layout resizable, expandable or collapsible</li>
+    </ul>
+  </Use>
+</Usage>
+
+## How to use it
+
+### Vertical
+
+Panes can be used in a vertical orientation. The panes resize up to the values
+that you set.
+
+import PaneVertical from '../../examples/pane/PaneVertical.tsx';
+import PaneVerticalCode from '!!raw-loader!../../examples/pane/PaneVertical.tsx';
+
+<CodeExample code={PaneVerticalCode}>
+  <PaneVertical />
+</CodeExample>
+
+### Horizontal
+
+Panes can be used in a horizontal orientation as well.
+
+import PaneHorizontal from '../../examples/pane/PaneHorizontal.tsx';
+import PaneHorizontalCode from '!!raw-loader!../../examples/pane/PaneHorizontal.tsx';
+
+<CodeExample code={PaneHorizontalCode}>
+  <PaneHorizontal />
+</CodeExample>
+
+### Grid
+
+Panes can be resized in both vertical and horizontal directions for a complex
+grid flexible layout.
+
+import PaneGrid from '../../examples/pane/PaneGrid.tsx';
+import PaneGridCode from '!!raw-loader!../../examples/pane/PaneGrid.tsx';
+
+<CodeExample code={PaneGridCode}>
+  <PaneGrid />
+</CodeExample>
+
+## Customization
+
+### Vertical collapse
+
+Panes can be collapsed or expanded by using the splitter together with an
+[Icon button](/components/icon-button). Vertical orientation uses a small circular
+icon button positioned on top of the splitter by default in both collapsed and
+expanded states. [Tooltips](/components/tooltip) should be used to describe the action.
+
+import PaneVerticalCollapse from '../../examples/pane/PaneVerticalCollapse.tsx';
+import PaneVerticalCollapseCode from '!!raw-loader!../../examples/pane/PaneVerticalCollapse.tsx';
+
+<CodeExample code={PaneVerticalCollapseCode}>
+  <PaneVerticalCollapse />
+</CodeExample>
+
+### Horizontal collapse
+
+Horizontal orientation also uses a small [Icon button](/components/icon-button) to
+collapse the pane. When the pane is collapsed, a stretched [Button](/components/button)
+is used to indicate an expandable state of the pane.
+
+import PaneHorizontalCollapse from '../../examples/pane/PaneHorizontalCollapse.tsx';
+import PaneHorizontalCollapseCode from '!!raw-loader!../../examples/pane/PaneHorizontalCollapse.tsx';
+
+<CodeExample code={PaneHorizontalCollapseCode}>
+  <PaneHorizontalCollapse />
+</CodeExample>
+
+### Icon button position
+
+Icon button can be positioned at the start, center or on the end of the splitter.
```

Comment:
It would be helpful if the pane content below actually guided the user to understanding what is what. I have a hard time telling at a glance what is going on. For example, the headings might read *Start*, *Center*, *End* (or something like that).

Reply:
Same as above, the "Icon button" should be updated to match the agreed-upon term (Action?). Will let @diana-oum weigh in here.

Agreed on the headings suggestion. Since all examples are combined, is it OK they just separate the examples? That way the content within the examples remains consistent. 

![image](https://user-images.githubusercontent.com/773721/181718847-db895406-704e-4d8e-aa4e-a73b7dac3d3c.png)

Reply:
Updated in 27aff6b32d3fdb88b5ead08650b7f09406f90ec8.